js/main.js

```diff
@@ -18,16 +19,42 @@ function submitForm(event) {
     notes: formElements.elements.notes.value,
     entryId: data.nextEntryId,
   };
-  data.nextEntryId++;
-  data.entries.unshift(formEntries);
-  imgElement.src = '../images/placeholder-image-square.jpg';
-  form.reset();
-  const unorderedList = document.querySelector('ul');
-  unorderedList.appendChild(renderEntry(formEntries));
-  viewSwap('entries');
-  if (localStorage.getItem('form-submission') === null) {
-    toggleNoEntries();
+
+  if (data.editing === null) {
+    data.nextEntryId++;
+    data.entries.unshift(formEntries);
+    imgElement.src = '../images/placeholder-image-square.jpg';
+    form.reset();
+    const unorderedList = document.querySelector('ul');
+    unorderedList.appendChild(renderEntry(formEntries));
```

Comment:
instead of `appendChild` there is another method you can use to insert this rendered entry to the beginning of the list:

[https://developer.mozilla.org/en-US/docs/Web/API/Element/prepend](url)